src/ui/ResultList/ResultList.ts

```diff
@@ -496,7 +501,10 @@ export class ResultList extends Component {
   }
 
   protected triggerNewResultsDisplayed() {
-    $$(this.element).trigger(ResultListEvents.newResultsDisplayed, {});
+    const args: IDisplayedNewResultsEventArgs = {
```

Comment:
I would just pass the object directly in the method call this way you don't need to define a variable + the interface. If you do it on multiple lines it's as readable and clear as here.